### PR TITLE
LPS-175676, LPS-175678, LPS-175679, LPS-175681, LPS-175691, LPS-175689 | Test Automation | Master

### PR DESCRIPTION
**Pull request by davisaints**
- [LPS-175676](https://issues.liferay.com/browse/LPS-175676)
- [LPS-175678](https://issues.liferay.com/browse/LPS-175678)
- [LPS-175679](https://issues.liferay.com/browse/LPS-175679)
- [LPS-175681](https://issues.liferay.com/browse/LPS-175681)
- [LPS-175691](https://issues.liferay.com/browse/LPS-175691)
- [LPS-175689](https://issues.liferay.com/browse/LPS-175689)

### CODE REVIEW

**Comment by davisaints** [comment]
ci:test:sf

**Comment by davisaints** [comment]
ci:test:sf

**Comment by timpak** [comment]
Taking a look

**Comment by davisaints** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/dependencies/simple_picklist.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+	"externalReferenceCode": "20eb955a-b05b-1a54-4ff6-62908b7dcbd8",
+	"listTypeEntries": [
+	],
+	"name": "Simple Picklist",
+	"name_i18n": {
+		"en-US": "Simple Picklist"
+	}
+}
```
Comment (andreefarias) [inline portal-web/test/functional/com/liferay/portalweb/dependencies/simple_picklist.json:9]: Add a `.json` to import Simple Picklist with items.

**Comment by davisaints** [comment]
ci:test:sf

File: modules/apps/object/object-test/src/testFunctional/tests/PicklistExportImport.testcase [hunk 2]
```diff
@@ -66,9 +66,61 @@ definition {
...
+
+			ObjectAdmin.addObjectFieldViaUI(
+				fieldLabel = "Field Picklist",
+				fieldPicklist = "ImportedPicklist175689",
+				fieldType = "Picklist");
+
+			ObjectAdmin.publishObjectViaAPI(objectName = "CustomObject175689");
+		}
+
+		task ("When translating the picklist definition") {
```
Comment (andreefarias) [inline modules/apps/object/object-test/src/testFunctional/tests/PicklistExportImport.testcase:200]: `task ("And when translating the picklist definition and item")`

File: modules/apps/object/object-test/src/testFunctional/tests/PicklistExportImport.testcase [hunk 3]
```diff
@@ -66,9 +66,61 @@ definition {
...
+			Picklist.gotoPicklists();
 
+			Picklist.gotoPicklistView(picklistName = "ImportedPicklist175689");
+
+			Picklist.addLanguageOnPicklist(
+				itemName = "PicklistImportada175689",
+				newLanguage = "pt_BR");
+		}
+
```
Comment (andreefarias) [inline modules/apps/object/object-test/src/testFunctional/tests/PicklistExportImport.testcase:209]: Add translation to Picklist Item too, this way it's possible to assert the item translated.

File: modules/apps/object/object-test/src/testFunctional/tests/PicklistExportImport.testcase [hunk 4]
```diff
@@ -66,9 +66,61 @@ definition {
...
+				languageName = "português (Brasil)",
+				userScreenName = "test");
+		}
+
+		task ("Then the user can add an object entry") {
+			ObjectAdmin.goToCustomObject(objectName = "CustomObject175689");
+
+			ObjectAdmin.addObjectPicklistFieldEntryViaUI(
+				picklistOption = "Picklist Item 1",
+				saveName = "Salvar");
```
Comment (andreefarias) [inline modules/apps/object/object-test/src/testFunctional/tests/PicklistExportImport.testcase:223]: Add picklist item translated

File: modules/apps/object/object-test/src/testFunctional/tests/PicklistExportImport.testcase [hunk 5]
```diff
@@ -66,9 +66,61 @@ definition {
...
+
+			ObjectAdmin.goToCustomObject(objectName = "CustomObject175689");
+
+			ObjectPortlet.viewEntry(entry = "Picklist Item 1");
```
Comment (andreefarias) [inline modules/apps/object/object-test/src/testFunctional/tests/PicklistExportImport.testcase:227]: Assert that item was translated.

**Comment by davisaints** [comment]
ci:test:sf

**Review by andreefarias — APPROVED** [review]

**Comment by andreefarias** [comment]
ci:forward

**Comment by andreefarias** [comment]
ci:stop

**Comment by andreefarias** [comment]
ci:forward

**Comment by andreefarias** [comment]
ci:stop

**Comment by andreefarias** [comment]
ci:forward

**Comment by andreefarias** [comment]
ci:forward

**Pull request closed without merging**